File: apps/problems/max-element/boilerplate-full/function.js
##USER_CODE_HERE##
const input = require("fs")
    .readFileSync("/dev/stdin", "utf8")
    .trim()
    .split("\n");
const n = parseInt(input.shift());
const arr = input.shift().split(" ").map(Number);

console.log(findMaxElement(n, arr));

File: apps/problems/max-element/boilerplate-full/function.cpp
#include <iostream>
#include <vector>
#include <string>
using namespace std;

##USER_CODE_HERE##

    int
    main()
{
  int n;
  cin >> n;
  vector<int> arr(n);
  for (int i = 0; i < n; ++i)
    cin >> arr[i];

  cout << findMaxElement(n, arr);
  return 0;
}